Userland/Libraries/LibJS/Runtime/Temporal/Calendar.cpp

```diff
@@ -30,10 +31,24 @@ Calendar::Calendar(String identifier, Object& prototype)
 {
 }
 
-// 12.1.1 CreateTemporalCalendar ( identifier [ , newTarget ] ), https://tc39.es/proposal-temporal/#sec-temporal-createtemporalcalendar
+// 12.1.1 IsBuiltinCalendar ( id ), https://tc39.es/proposal-temporal/#sec-temporal-isbuiltincalendar
+bool is_builtin_calendar(String const& identifier)
+{
+    // 1. Let calendars be AvailableCalendars().
+    auto calendars = Unicode::get_available_calendars();
+
+    // 2. If calendars contains id, return true.
+    if (calendars.contains_slow(identifier))
+        return true;
+
+    // 3. Return false.
+    return false;
+}
```

Comment:
In serenity terms this is not just an editorial change, it's a normative one as well, since we suddenly support IDs other than iso8601 now. Are we sure the rest of the exposed Temporal APIs protected by this API are ready to handle them?

Reply:
e.g. a lot of the methods in Temporal marked with `NOTE: This is the minimum XXXX implementation for engines without ECMA-402.` assert that the calendar id is `iso8601`

Reply:
Very good point, fixed :+1: